### PR TITLE
Adding GLUT Shapes support

### PR DESCRIPTION
**Pull request by iyenal**
PSPGL is missing GLUT shapes support, so this change adds it.
More details on supported shapes on https://github.com/iyenal/PSPGL-OpenGL-Shapes

### CODE REVIEW

File: pspgl/glut.c [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include <pspkerneltypes.h>
 #include <pspctrl.h>
 #include <psploadexec.h>
+#include "glut_shapes.c" // Disable this if you don't want GLUT shapes support
```
Comment (dbeef) [inline pspgl/glut.c:8]: `.c` files are not really intended for including ;)

Comment (dbeef) [reply pspgl/glut.c:8]: Remove this include, and instead, in `/pspgl/Makefile`, 
append `glut_shapes.c` to glut target, like this:

```makefile
libglut.a_OBJS = \
	glut.o
        glut_shapes.o
```

**Comment by iyenal** [comment]
Sure thing

**Comment by iyenal** [comment]
Should be all good now

**Pull request merged**